app/components/degree_row_content.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class DegreeRowContent < ViewComponent::Base
```

Comment:
Is it worth putting all of these in a `Publish` folder to help keep them separate, since we have a folder for Find?

Reply:
Eventually i think we should do that for as many files as possible to create boundaries but can leave it for another ticket

Reply:
We are leaving the multi nested folders for now, there is some higher priority stuff to get on with....the app/components/foo.rb went back and forth for a bit, settled on present structure.